src/functionapp/ai_ocr/azure/openai_ops.py

```diff
@@ -33,10 +36,3 @@ def get_size_of_base64_images(images):
     for img in images:
         total_size += len(img)
     return total_size
-
-
-load_image_chain = TransformChain(
```

Comment:
I could not find any place where this chain was referenced or used, so I removed it. If this was necessary for a larger piece of the solution, let me know so that I can be sure to include a Semantic Kernel replacement.